modules/validation/binding_test.go

```diff
@@ -26,9 +26,10 @@ type (
 	}
 
 	TestForm struct {
-		BranchName  string `form:"BranchName" binding:"GitRefName"`
-		URL         string `form:"ValidUrl" binding:"ValidUrl"`
-		GlobPattern string `form:"GlobPattern" binding:"GlobPattern"`
+		BranchName   string `form:"BranchName" binding:"GitRefName"`
+		URL          string `form:"ValidUrl" binding:"ValidUrl"`
+		GlobPattern  string `form:"GlobPattern" binding:"GlobPattern"`
+		RegexPattern string `form:"GlobPattern" binding:"RegexPattern"`
```

Comment:
why do we need a RegexPattern if we use Glob?

Reply:
The binding name is a copy paste error but this PR supports both, Regex and Glob, as tag name pattern.